download.js

```diff
@@ -558,8 +560,8 @@ function generateCode(){
 }
 
 function buildCode(promises) {
-	var i = 0,
-	    l = promises.length;
+	var i = 0;
+	var l = promises.length;
```

Comment:
These files all have changes which add `var` statements, despite the prism-core files all allowing multiline vars. What's the story here?

Reply:
This is because of [`no-mixed-spaces-and-tabs`](https://eslint.org/docs/rules/no-mixed-spaces-and-tabs) (see commit https://github.com/PrismJS/prism/pull/1796/commits/53a54b40a433ca390918a7d3d0514718fc438ac8).

I’ll probably use the `smart-tabs` option which I didn’t know about until now.

Reply:
[Here](https://github.com/PrismJS/prism/pull/1796#pullrequestreview-212046340)'s the story.

The problem is that tabs followed by spaces are considered mixed indentation and some auto formatter just replace the 4 spaces with a tab. Most of Prism core has already fallen victim to these formatters. But instead of saving the wounded I think it's easier to just finish them off. (Enough metaphors for today)

The idea is that if you just write the additional `var`s you don't have these problems (mixed indentation, war against formatters).